### DISCUSSION TITLE
Comfyui .bat files not responding

### DISCUSSION THREAD

**Discussion by Booska-Boo** [Comfy-Org/ComfyUI#2957] [Q&A]
I'm in the process of installing Comfyui according to the instructions on Monzon Media's YouTube channel. But nothing happens when I try to run the .bat file, run_nvidia_gpu. The run_cpu .bat file doesn't work either. I've been trying to figure out why they don't work, but haven't seen much about this problem on the message boards.

Has anyone else run into this issue and figured it out? Thanks in advance.

**Comment by AlistairJon** [comment]
I'm having a similar problem installing Comfyui. All protection  shields are off.  "The system cannot find the path specified." Any information / help would be appreciated.

**Reply by AlistairJon** [reply]
Sorry. I should elaborate. Both bat files as with Booska-Boo above appear to load prior to opening; but cmd gives the message "the system cannot find the path specified."

**Reply by LukeG89** [reply]
@AlistairJon Are you running the .bat files as administrator? Because you just have to double-click them to run

**Comment by AlistairJon** [comment]
Thank you for the reply - yes I am running the .bat files as administrator.

**Reply by LukeG89** [reply]
That's the problem, you just have to double-click it

**Reply by catman133** [reply]
I have the exact same problem as Alistair i dubble click the bat file but cmd tells me the system cannot find the path specified anyone know how to fix this
